### ISSUE TITLE
Use of Nim's standard lib colors implementation

### ISSUE THREAD

**Issue by Jonathan-Racaud** [trustable-code/NiGui#99]
Hi,

You are defining a Color object, but Nim already defines a Color type in its standard library:
[color.nim](https://github.com/nim-lang/Nim/blob/master/lib/pure/colors.nim)

Wouldn't using that instead of redefining the type be possible ?

Plus it would make working with Colors easier as the nim implementation has more features and that would avoid managing different types for the same goal.

The only problem that I see though is that the std's implementation doesn't seem to handle the alpha channel (maybe that's the reason for not using the Nim's implementation ?).

**Comment by trustable-code** [comment]
You're right, it would be nice to have one common Color type, but handling of alpha channel needs to be ensured.
